doc/user/content/manage/user-management/_index.md

```diff
@@ -0,0 +1,130 @@
+---
+title: "Role based access control"
+description: "Manage roles, permissions, and other access control options in Materialize"
+disable_list: true
+menu:
+  main:
+    parent: manage
+    name: User management
+    identifier: 'user-management'
+    weight: 10
+---
+
+{{< alpha />}}
+
+This page introduces role-based access management (RBAC) in Materialize. RBAC
+allows you to apply granular privileges to your Materialize objects and clusters. Organizations
+using RBAC can manage user roles and privileges to ensure there is not
+unauthorized or improper access to sensitive objects.
+
+In Materialize, RBAC allows organization administrators to:
+
+* Determine which users have read or write permissions for specific objects
+
+* Control how users interact with clusters by giving them different levels of access to
+resources
+
+* Prevent accident operations from unauthorized users
+
+* Isolate access to user-facing data from internal organization data
+T
+Materialize object access is also dependent on cluster privileges.
+Roles that need access to an object that use compute resources must also have
+the same level of access to the cluster. Materialize objects that use compute
+resources are:
+
+* Replicas
+* Sources
+* Sinks
+* Indexes
+* Materialized views
+
+he next sections go over the concepts of authorization and authentication and
+the objects within Materialize.
+
+## Authentication vs. authorization
+
+Authentication dictates who can log in to a system while authorization
+determines what a user can access within a system.
+
+Authentication in Materialize is handled when you sign up or are invited to a
+Materialize organization. Your immediate privileges are determined by your
+invitation from an administrator or are assigned automatically by Materialize
+when you sign up.
+
+Authorization is determined in RBAC by your organization administrator. When you
+invite users to your Materialize organization, you have the option to give the
+user more elevated privileges in the Materialize administrator console.
+
+## RBAC structure
+
+RBAC in practice is a group of roles with assigned privileges and attributes.
+You can assign specific users to roles or assign privileges to users to inherit
+from other roles.
+
+### Roles
+
+A role is a collection of permissions you can apply to users. Roles make it
+easier to assign or revoke privileges on Materialize objects. You can group
+users into specified roles with different levels of permissions and adjust those
+permissions to ensure they have the correct level of access to objects.
+
+### Role attributes
+
+Role attributes are actions available to any role you create. Attributes are
+independent of any other object in Materialize and apply to the entire
+organization. You can edit these actions when you create the role:
+
+| Name            | Description                                                                     |
```

Comment:
I see other attributes in the `CREATE ROLE` doc: here: https://materialize.com/docs/sql/create-role/#syntax. Shouldn't they all be covered here as well?